### PR TITLE
Addition of redirects and robots txt

### PR DESCRIPTION
**Pull request by Kameshwaran**
### This PR contains,

1. Added necessary redirects for missing pages
2. Added a robots.txt file to inform google crawler to not crawl the missing links further.
3. Added logic to redirect missing pages to '/'
4. Added command to copy robots.txt to the public directory after building

_Note:_ 
Missing pages mean the pages which are available on the old website, but not created on the new website.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
   ],
   "license": "MIT",
   "scripts": {
-    "build": "gatsby build",
+    "build": "gatsby build && cp robots.txt public/robots.txt",
```
Comment (sanandnarayan) [inline package.json]: @Kameshwaran a better way to do this is
https://www.gatsbyjs.org/docs/static-folder/

**Review by sanandnarayan — CHANGES_REQUESTED** [review]
please move robots.txt to static folder

File: package.json (hunk 1 above)
Comment (Kameshwaran) [reply package.json]: good catch. I wonder how i forget this basic stuff. I've already done this for static scripts

**Comment by Kameshwaran** [comment]
@sanandnarayan done with the changes.

**Review by sanandnarayan — APPROVED** [review]

**Pull request merged**